openid/openid.go

```
@@ -240,9 +255,24 @@ func (c *Client) Verify(requestURL string) (*Response, error) {
 			r.SReg[k[len("openid.sreg."):]] = v.Get(k)
 		}
 	}
+	if v.Get("openid.ns.macaroon") == nsMacaroon && contains(signed, "macaroon.discharge") {
+		r.Discharge, _ = decodeMacaroon(v.Get("openid.macaroon.discharge"))
```

Perhaps add a comment explaining why it's OK to ignore the error here?